tests/unit/test_self_check_outdated.py

```diff
@@ -59,13 +59,13 @@ def test_pip_self_version_check_calls_underlying_implementation(
 
 
 @pytest.mark.parametrize(
-    [
-        "installed_version",
-        "remote_version",
-        "stored_version",
-        "installed_by_pip",
-        "should_show_prompt",
-    ],
+    (
+        "installed_version, "
+        "remote_version, "
+        "stored_version, "
+        "installed_by_pip, "
+        "should_show_prompt"
+    ),
```

Comment:
This is a bit counter-productive tbh but I guess the linter needs to have _one_ rule.

Reply:
Well, this was because of the spaces, the string exceeded the max line length here, so I can remove the spaces, or break it up some other way, or use a `# noqa`, or change the max line length for the test directory.

Happy to go with a any solution.

Reply:
Updated this to a `noqa: PT006` so it doesn't become more difficult to read